sound/soc/sof/sof-priv.h

```diff
@@ -431,10 +431,10 @@ struct snd_sof_dev {
 	int dma_trace_pages;
 	wait_queue_head_t trace_sleep;
 	u32 host_offset;
-	u32 dtrace_is_supported; /* set with Kconfig or module parameter */
-	u32 dtrace_is_enabled;
-	u32 dtrace_error;
-	u32 dtrace_draining;
+	bool dtrace_is_supported; /* set with Kconfig or module parameter */
+	bool dtrace_is_enabled;
+	bool dtrace_error;
+	bool dtrace_draining;
```

Comment:
all of those were to avoid checkpatch warnings on using bool in structures.

Reply:
@lyakh Any comment on this? checkpatch seems now ok with this...?